assets/js/modules/ads/components/settings/SettingsView.stories.js

```diff
@@ -112,9 +112,7 @@ export default {
 
 export const PaxConnected = Template.bind( null );
 PaxConnected.storyName = 'With PAX onboarding';
-PaxConnected.scenario = {
-	label: 'Modules/Ads/Settings/SettingsView/PAX',
-};
+PaxConnected.scenario = {};
```

Comment:
Since the label property was removed, a new set of VRT images have been added. Can you please remove the old images?